.travis.yml

```diff
@@ -1,3 +1,4 @@
 language: java
+dist: trusty
```

Comment:
Did you get a chance to look at this? https://talk.openmrs.org/t/travis-ci-error-expected-feature-release-number-in-range-of-9-to-14-but-got-8/25238/7

Reply:
No, I missed that. I will push a commit(ammend the last commit) to use openjdk instead. 

These were the two options provided by `Travis-CI` people and I thought changing the `jdk` distributor might be a big call.

Reply:
@dkayiwa Done. Thanks for pointing out to the right thread.